aiohttp/web_response.py

```diff
@@ -53,7 +53,7 @@
     BaseClass = collections.abc.MutableMapping
 
 
-class ContentCoding(enum.Enum):
+class ContentCoding(enum.StrEnum):
```

Comment:
Okay not sure I need this anymore, but I would need to dumb down the dict type otherwise.  I'll check.

Reply:
If you think the code is better with a newer Python feature, then ideally add 2 versions with a `if sys.version_info >= (3, 10)`, or if too complicated, just leave a `# TODO (PY310): ...` comment. Then we'll drop the older code as we upgrade in future.

Reply:
Personally, I'm a stickler for strong typing, so I would prefer to keep it instead of dumbing things down.  And I see spots in the code where it could be used more, especially if I try to follow this up with dynamic Brotli compression.

If you all are okay with it, I'd do an if-block just around the import for 3.11, and use either [`strenum`](https://pypi.org/project/StrEnum/#description) or [`backports.stremum`](https://pypi.org/project/backports.strenum/).

If so, a hint on how to easily add the dep in the right spot would be appreciated.  My python is a bit rusty.

Reply:
With the enum changes in python 3.11 I would prefer to keep this as is  since it introduces additional risk.

I have had hard to find problems with Enum related changes in cpython 3.11 and would prefer to be cautious here.

Once we drop support for python less than 3.11 we can come back and rework this.

Reply:
Okay I removed the `StrEnum` while keeping the literal typing and use of the existing `Enum`.  Should pass tests now.

Reply:
Literal is unfortunately not available on all python versions we currently support.

Reply:
Yeah I was going to say I checked the 3.8 docs first... need to put my kids to bed then I'll finish this up.

Reply:
Finally fixed - I think 🤞🏻